src/redux/reducer.js

```diff
@@ -211,6 +211,7 @@ const initialState = {
     authenticationRouterError: null,
     showAuthenticationRouterLogin: false,
     studyUpdated: { force: 0, eventData: {} },
+    mapDataLoading: true,
```

Comment:
I would have set it to false by default

Reply:
Actually it's impossible for the user to modify the node before the load of map data => I changed the initialization value to true